builds/e2e/templates/e2e-run.yaml

```diff
@@ -46,8 +46,16 @@ steps:
       {
         $filter += '&Name~TempSensor'
       }
-
-      sudo --preserve-env dotnet test $testFile --no-build --logger 'trx' --filter "$filter"
+      
+      #Dotnet SDK 3.1.415 package on Centos doesn't allow dotnet to be accessed via sudo command due to Path issues. Use the below workaround for centos only.
+      if ('$(artifactName)'.Contains('centos'))
+      {
+        sudo --preserve-env $(command -v dotnet) test $testFile --no-build --logger 'trx' --filter "$filter"
```

Comment:
Is this going to sometimes run on a Windows agent? If so, then `command -v dotnet` won't work. We should use the equivalent PowerShell command instead: `(Get-Command dotnet).Source`.

Reply:
No, it's only supposed to run on CentOS. For all other platforms, we use the default dotnet command

Reply:
🙄 I should have seen that. Thanks!